orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/tasks/DisableInstancesTask.groovy

```diff
@@ -47,6 +47,7 @@ class DisableInstancesTask implements CloudProviderAware, Task {
     String account = getCredentials(stage)
 
     def serverGroupName = stage.context.serverGroupName ?: stage.context.asgName
+    def interestingHealthProviderNames = HealthHelper.getInterestingHealthProviderNames(stage, ["LoadBalancer"])
```

Comment:
Most stages aren't going to have `interestingHealthProviderNames` explicitly defined right?

With this change I feel we're going to miss out on Discovery in almost all cases right?

We do have the `DetermineHealthProvidersTask` but it's provider-specific and doesn't look at underlying account attributes.